lib/history.js

```diff
@@ -143,7 +143,8 @@ function didClick (router, event) {
   while (target && target.nodeName.toLowerCase() !== 'a' && target !== html) {
     target = target.parentNode
   }
-  if (!target || target.nodeName.toLowerCase() !== 'a') return
+  if (!target || target.nodeName.toLowerCase() !== 'a' ||
+      target.hasAttribute('target') || target.hasAttribute('download')) return
```

Comment:
can you add the `rel` attribute as well? `rel=external` is a valid way to say the link leaves site.